src/app/moods/info/page.tsx

```diff
@@ -1,3 +1,164 @@
-export default function Page() {
-  return <h1>this is a placeholder</h1>;
+"use client";
+
+import Image from "next/image";
+import Link from "next/link";
+import { Header } from "@/ui/shared/Header";
+import Toggle from "@/ui/shared/Toggle";
+import { useState } from "react";
+
+const tasks = [
+  {
+    title: "Urgent & Important",
+    description: "Do these tasks immediately (e.g., deadlines, emergencies).",
+  },
+  {
+    title: "Not Urgent but Important",
+    description: "Schedule these tasks (e.g., planning, exercise).",
+  },
+  {
+    title: "Urgent but Not Important",
+    description: "Delegate these tasks if possible (e.g., interruptions).",
+  },
+  {
+    title: "Not Urgent & Not Important",
+    description: "Eliminate or minimize these tasks (e.g., distractions).",
+  },
+];
+
+const neuroInfo = [
+  {
+    title: "Dopamine",
+    description: "(Motivation)",
+  },
+  {
+    title: "Norepinephrine",
+    description: "(Intensity)",
+  },
+  {
+    title: "Serotonin",
+    description: "Emotional stability",
+  },
+];
+
+export default function MoodsInfoPage() {
+  const [isPriority, setIsPriority] = useState(false);
+
+  return (
+    <>
+      <Header title="Learn More" isInfoPage={true} />
+      <Toggle
+        isToggled={isPriority}
+        setIsToggled={setIsPriority}
+        toggledOff={"Mood"}
+        toggledOn={"Priority"}
+        showLabels={true}
+      />
+      <div className="w-10/12 m-auto flex flex-col gap-10 mt-8">
+        {!isPriority ? (
```

Comment:
```suggestion
        {isPriority ? (
```
Currently the info pages are flipped